stor/dx.py

```diff
@@ -56,38 +62,73 @@ class ProjectNotFoundError(NotFoundError):
     pass
 
 
+class InconsistentUploadDownloadError(DNAnexusError):
+    """Thrown during checksum mismatch or part length mismatch.."""
+    pass
+
+
+def _dx_error_to_descriptive_exception(client_exception):
+    """Converts dxpy errors to more descriptive exceptions with
+    transaction ID"""
+    http_status = getattr(client_exception, 'code', None)
+    if isinstance(client_exception, dxpy.DXAPIError):
+        exc_str = '{} - {}'.format(client_exception.name or '',
+                                   client_exception.msg or client_exception.message or '')
+    else:
+        exc_str = str(client_exception)
+    status_to_exception = {
+        401: stor_exceptions.UnauthorizedError,
+        403: stor_exceptions.UnauthorizedError,
+        404: stor_exceptions.NotFoundError,
+        409: stor_exceptions.ConflictError
+    }
+    if http_status in status_to_exception:
+        logger.error('error in dxpy operation - %s', exc_str)
```

Comment:
please remove this logger.error.  There should be *no* error logging in this function.

Reply:
Removed.